public/js/ConversationController.js

```diff
@@ -46,4 +46,16 @@
         var id = setInterval(updateConversation, 1000);
 
     }]);
+
+    app.directive("chatAppConversation", function() {
```

Comment:
I don't know, the directive uses the controller and is pretty small so it made sense to me to put it in the same file, but if it's standard practice to put it in its own file I'll do that.